### PR TITLE
Removed whitespaces and comments from orchestrator script

### PR DESCRIPTION
**Pull request by khanguslee**
No more commented out lines or whitespaces pls

### CODE REVIEW

**Comment by khanguslee** [comment]
Also changed it so we send the power model plan name when we start the power model and we only send power plan generated when we generate a power plan. Sending a power plan name is also a retained message so that ANY client that subscribes to that topic (before/after publishing this message) will receive the plan name that is currently used for power model.

This should also be set later on when we select a power plan from the website.

File: orchestra.py [hunk 1]
```diff
@@ -34,7 +32,7 @@ def on_connect(client, userdata, flags, rc):
     client.subscribe("power_model/start")
```
Comment (harsilspatel) [inline orchestra.py]: do we also want to integrate the topics.py?

Comment (khanguslee) [reply orchestra.py]: Feel free to make a PR 💯

File: orchestra.py [hunk 2]
```diff
@@ -48,77 +46,58 @@ def parse_data(data):
...
-        #json_file = parsed_data["json_file"]
-        
+
         interpolation_path = os.path.join(os.path.dirname(os.path.realpath(sys.argv[0])), "interpolation.py")
         arguments = ["python3", interpolation_path]
         output_interpolation = subprocess.check_output(arguments, stderr=subprocess.STDOUT)
-        
+
         rider_plan_finder_path = os.path.join(os.path.dirname(os.path.realpath(sys.argv[0])), "rider_plan_finder.py")
         arguments = ["python3", rider_plan_finder_path,"--json_file",json_file_name]
```
Comment (harsilspatel) [inline orchestra.py]: i don't really like the underscores here, maybe i'll update them in the next PR :)

Comment (khanguslee) [reply orchestra.py]: Yeah underscore for arguments are a nono

**Comment by khanguslee** [comment]
> minor tweaks needed which i'll take care of next week, good stuff @khanguslee!!

Since I was working on the code, I might as well make changes now rather than later. @chamaka1 pls review this PR

File: orchestra.py (hunk 2 above)
Comment (harsilspatel) [reply orchestra.py]: @chamaka1 *ahem 😛

**Pull request merged**